File: Python/Assignment-2/Prog6.py
marks = int(input("Enter your marks : "))
if(marks>=90):
    print("Grade : Ex")
elif(marks>=80 and marks<90):
    print("Grade : A")
elif(marks>=70 and marks<80):
    print("Grade : B")
elif(marks>=60 and marks<70):
    print("Grade : C")
elif(marks>=50 and marks<60):
    print("Grade : D")
elif(marks>=35 and marks<50):
    print("Grade : P")
else:
    print("Grade : F")

File: C/Assignment-2/Prog6.c
#include<stdio.h>

int main(){
    int marks = 0;
    scanf("%d", &marks);
    
    if (marks>=90 && marks<=100)
    {
        printf("Grade : Ex");
    }
    else if (marks>=80 && marks<90)
    {
        printf("Grade : A");
    }else if (marks>=70 && marks<80)
    {
        printf("Grade : B");
    }else if (marks>=60 && marks<70)
    {
        printf("Grade : C");
    }else if (marks>=50 && marks<60)
    {
        printf("Grade : D");
    }else if (marks>=35 && marks<50)
    {
        printf("Grade : P");
    }else if (marks<35)
    {
        printf("Grade : F");
    }
    else
        printf("Invalid");
    return 0;
}